src/parsers/parseArrayToStringWithAticle.ts

```diff
@@ -0,0 +1,17 @@
+const parseArrayToStringWithArticle = (arr: []): String => {
+  let str = "";
+  let arrSize = arr.length;
```

Comment:
You don't need the `arr` prefix in this variable name. Since we're working with functions, small pieces of logic/code, we have the context in favor of nomenclature and becomes unnecessary to use prefix or large nomenclatures for functions, variables etc.